src/main/java/core/basesyntax/AreaCalculate.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculate {

```

Comment:
rename as `AreaCalculator`, the class name should be noun according the naming convention